test/screens/v2/widget_instance/reconstructed_alert_test.exs

```diff
@@ -1235,6 +1235,94 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlertTest do
                widget |> put_solo_screen() |> ReconstructedAlert.serialize()
     end
 
+    test "partial platform closure for here", %{widget: widget} do
+      widget =
+        widget
+        |> put_home_stop(PreFare, "place-jaksn")
+        |> put_effect(:station_closure)
+        |> put_informed_station_names(["Jackson Square"])
+        |> put_informed_entities([
+          ie(stop: "place-jaksn", route: "Orange", route_type: 1)
+        ])
+        |> put_tagged_stop_sequences(%{
+          "Red" => [["place-jaksn", "place-forhl"]]
```

Comment:
This seems like it should be `"Orange"` given that's the route being used elsewhere in this test case; not sure how it's passing anyway?